docs/site/Authentication-component-action.md

```diff
@@ -0,0 +1,252 @@
+---
+lang: en
+title: 'Concept Two - Authentication Action'
```

Comment:
Not sure if we need "concept one", "concept two". Personally I'd prefer not to. 
If the parent page mention there are 3 concepts you need to know for authentication in LoopBack, and list the three with links. I think this would be good.

Reply:
I was trying to distinguish them from the "options" section cuz option is not really a concept. Hmm, maybe rename them to 

- Authentication Decorator
- Authentication Action
- Authentication Strategy
- Configuring Strategy Options

?